scripts/load_meeting_prot.py

```diff
@@ -21,91 +24,90 @@ def get_ws(wsheet_title,ssheet):
+                proj_name = row_list[0]
+                info = ' '.join(row_list[1:])
+                if not proj_name in dict_holder.keys():
+                    dict_holder[proj_name] = {'info' : info, 'flowcells' : {}}
+            elif proj_name and not row[0].isdigit():
+                if dict_holder[proj_name].has_key('info'):
+                    if row not in dict_holder[proj_name]['info']:
+                        info = dict_holder[proj_name]['info']
+                        more_info = ' - '.join([info, row])
+                        dict_holder[proj_name]['info'] = more_info
+            elif proj_name and row[0].isdigit():
+                if not dict_holder[proj_name]['flowcells'].has_key(row):
+                    dict_holder[proj_name]['flowcells'][row] = []
+                proj_name = None
     return coming_deliveries, ongoing_deliveries
 
 def update(sorted_names, col, info, ss_key, ws_key):
     """Uppdates the new meeting protocol with old and new info."""
-    i=2
-    for name in sorted_names:
-        name = name[0]
-        CLIENT.UpdateCell(i, col, name , ss_key, ws_key)
-        CLIENT.UpdateCell(i, col+1, info[name]['info'] , ss_key, ws_key)
-        fcs = sorted(info[name]['flowcells'].keys())
-        for fc in fcs:
-            i=i+1
-            j=i
-            CLIENT.UpdateCell(i, col+1, fc, ss_key, ws_key)
-            for comment in info[name]['flowcells'][fc]:
-                CLIENT.UpdateCell(j, col+2, comment, ss_key, ws_key)
-                j=j+1
-            if i!=j:
-                i=j-1
-        i=i+2
+    row = 2
+    for proj_name in sorted_names:
+        proj_name = proj_name[0]
+        CLIENT.UpdateCell(row, col, proj_name , ss_key, ws_key)
+        CLIENT.UpdateCell(row, col+1, info[proj_name]['info'] , ss_key, ws_key)
+        sorted_fcs = sorted(info[proj_name]['flowcells'].keys())
+        for fc in sorted_fcs:
+            row += 1
+            comments_row = row
+            CLIENT.UpdateCell(row, col+1, fc, ss_key, ws_key)
+            for comment in info[proj_name]['flowcells'][fc]:
+                CLIENT.UpdateCell(comments_row, col+2, comment, ss_key, ws_key)
+                comments_row += 1
+            if row != comments_row:
+                row = comments_row - 1
+        row += 2
 
 def main(old_wsheet, new_wsheet, file, ill_fix_it_myself):
-    if os.path.exists("tempo.txt"): os.system("rm tempo.txt")
     if ill_fix_it_myself:
-        old_wsheet = date.today()-timedelta(days=7)
+        if os.path.exists("dump_from_trello.txt"):
+            os.system("rm dump_from_trello.txt")
+        old_wsheet = date.today() - timedelta(days = 7)
         old_wsheet = old_wsheet.isoformat()
         new_wsheet = date.today().isoformat()
-        file = "tempo.txt"
-        os.system("update_checklist.py hugin-conf.yaml >> tempo.txt")
+        file = "dump_from_trello.txt"
+        os.system("update_checklist.py hugin-conf.yaml >> dump_from_trello.txt")
```

Comment:
other thing (minor). It my be useful to store once per week the dum_from_trello file, so maybe instead of replacing it might be interesting to save it with a time stamp (dump_from_trello_YYMMDD)